ch01.asciidoc

```diff
@@ -85,6 +85,16 @@ A merchant will be happy to get an additional payment method which is virtually
 [[getting_first_bitcoin]]
 ==== Getting Your First Bitcoin on the Lightning Network
 
+Custodial Solutions
+
+Custodial solutions are where a 3rd party has ownership (custody) of your assets, and provides you a service based on this. Custodial solutions are familiar to anyone who has a bank account. In an ideal world, non-custodial lightning networs solutions would be easy enough to use that custodial solutions are not necessary, however in the current state of the crypto-currency lifecycle, there is a place for custodial solutions.  This book will not recommend any specific custodial solutions, however they may be a starting place for people wishing to experiement with the technology. 
```

Comment:
"easy enough" is quite judging / subjective. I'd rather have it more objective. 

I like the idea of not recommending a specific custodial solution (: